cmd/pack3d/main.go

```diff
@@ -1,33 +1,34 @@
 /*
 Instruction: write down this into the command line to use the software
-<pack3d {your_frame_size_X,your_frame_size_Y,your_frame_size_Z} spacing your_output_name STL_numbers STL_path STL_numbers STL_path .....>
-For example: <pack3d {100,100,100} 5 Pikacu 1 /home/corner.stl 2 /home/Pika.stl>
+<pack3d --build_volume=your_frame_size_X,your_frame_size_Y,your_frame_size_Z --json_file=json_config_file --spacing=spacing --filename=export_filename>
```

Comment:
`spacing` and `build_volume` need to end up in the json file.

Reply:
done!